software-update/fpga_list.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+ls /etc/fpga/
+mount /dev/sda1 /mnt
+mkdir /temp_fpga
+cp /mnt/*.jam /temp_fpga
+umount /mnt
```

Comment:
'The custom update files(.jam and .px4) are present in the connected storage media, which is mounted(Since automounting is absent) and the files are copied to the board(temporary location - temp_fpga and temp_fc_fm directory) for updation, and post that they are removed' - this is the approach that we had taken. Now that you are suggesting that no mounting, unmounting and USB connections are needed, could you please specify where will the custom files be present? If they will reside on the board, could you please specify the path?